src/components/pages/landing/workshop/WorkshopCard/WorkshopCard.tsx

```diff
@@ -1,11 +1,97 @@
+                        nesciunt repellat! Dolorem consectetur, delectus facere neque nesciunt suscipit! Exercitationem nobis ducimus veniam.
+                    </p>
+                </div>
+                <div className={s.infoWrap}>
+                    <div className={s.infoColumn}>
+                        <div className={s.info}>
+                            <span className={s.span}>
+                                <AlarmIcon className={s.icons} />
+                                Data i godzina
+                            </span>
+                            {workshop.startsAt}
+                        </div>
+                        <div className={s.info}>
+                            <span className={s.span}>
+                                <TimelapseIcon className={s.icons} />
+                                Czas trwania
+                            </span>
+                            {workshop.durationMinutes} min
+                        </div>
+                        <div className={s.info}>
+                            <span className={s.span}>
+                                <SchoolIcon className={s.icons} />
+                                Uczelnia
+                            </span>
+                            {workshop.university.name}
+                        </div>
+                        <div className={s.info}>
+                            <span className={s.span}>
+                                <PlaceIcon className={s.icons} />
+                                Miejsce warsztatu
+                            </span>
+                            {workshop.address}
+                        </div>
+                    </div>
+                    <div>
+                        <div className={s.info}>
+                            <span className={s.span}>
+                                <EmojiFlagsIcon className={s.icons} />
+                                Minimalny rok studiów
+                            </span>
+                            {workshop.minYearOfStudy}
+                        </div>
+                        <div className={s.info}>
+                            <span className={s.span}>
+                                <EmojiFlagsIcon className={s.icons} />
+                                Maksymalny rok studiów
+                            </span>
+                            {workshop.maxYearOfStudy}
+                        </div>
+                        <div className={s.info}>
+                            <span className={s.span}>
+                                <EmojiFlagsIcon className={s.icons} />
+                                Kierunek studiów
+                            </span>
+                            {workshop.preferableFieldsOfStudy.map((field) => field.name).join(", ")}
+                        </div>
+                    </div>
+                </div>
```

Comment:
Zauważ, że w sumie zawartość diva z klasa infoWrap jest bardzo powtarzalna. Spróbuj zmienić tą strukturę tak abyś mógł  stworzyć tablicę zawierającą obiekty z powtarzalnymi właściwościami i po niej tutaj mapować. Przykładowy obiekt w tablicy:
```
const WORKSHOP_DETAILS = [
{
   icon: <EmojiFlagsIcon />,
   detailName: "Maksymalny rok studiów"
}, ...
]
```

Trzymajmy się takiej konwencji, że jeśli potrzebujemy stałych używanych tylko w jednym  komponencie, twórzmy plik w tym przyapadku WorkshopCard.const.ts w którym umieścimy i wyeksportujemy tablicę. Następnie już w componencie importujemy tą tablicę i możemy w tym przypadku po niej mapować.

Jeśli chodzi o dynamiczną część każdego z info to nie powinniśmy jej dawać do const bo jest inna w zależności od ID workshopa ale nie jest to problem bo tych dynamicznych informacji  (maxYearOfStudy, minYearOfStudy itp...) jest tyle samo co tych stałych wiec będziesz mógł mógł sobie stworzyć jakąś fukcję, która stworzy ci tablice z tymi danymi warsztatu które tam potrzebujesz wypisać i potem w tym mapie odwołać się po indeksach, jakby mapowac jadną tablice ale równolegle też używać drugiej. 

W funkcji map: 
`someArray.map((someArrayElement, index) => ...) `
jest drugi argument index ktory ci na to pozwoli
Tutaj masz sprytny sposób na eksraktowanie interesujących cię właściwości z obiektu
`const { minYearOfStudy, maxYearOfStudy, address  ...} = workshop;`